app/org/sagebionetworks/bridge/play/controllers/ParticipantController.java

```diff
@@ -43,7 +43,7 @@ public Result getSelfParticipant() {
         UserSession session = getAuthenticatedSession();
         Study study = studyService.getStudy(session.getStudyIdentifier());
         
-        StudyParticipant participant = participantService.getParticipant(study, NO_CALLER_ROLES, session.getId());
+        StudyParticipant participant = participantService.getParticipant(study, session.getId(), false);
```

Comment:
It's marked @JsonIgnore, however, the encryptedHealthCode is being exposed. Let's get this PR fixed and merged and I'll create a follow-up PR to hide encryptedHealthCode in the API.